app/src/main/java/org/jellyfin/androidtv/ui/home/HomeFragment.kt

```diff
@@ -106,6 +109,28 @@ class HomeFragment : StdBrowseFragment(), AudioEventListener {
 			// Update the views before creating rows
 			views = callApi<ItemsResult> { apiClient.GetUserViews(TvApp.getApplication().currentUser!!.id, it) }
 
+			// Check for live TV support
+			if (TvApp.getApplication().currentUser!!.policy.enableLiveTvAccess) {
+				// This is kind of ugly, but it mirrors how web handles the live TV rows on the home screen
+				// If we can retrieve one live TV recommendation, then we should display the rows
+				callApi<ItemsResult> {
+					apiClient.GetRecommendedLiveTvProgramsAsync(
+						RecommendedProgramQuery().apply {
+							userId = TvApp.getApplication().currentUser!!.id
+							enableTotalRecordCount = false
+							fields = arrayOf(
+								ItemFields.ChannelInfo,
+								ItemFields.PrimaryImageAspectRatio
+							)
```

Comment:
Don't need the fields since it is only to check if there is any results

Reply:
I wasn't sure why web included them, but it doesn't seem to make any difference in my testing so I have removed them.